module/signature/threshold.go

```diff
@@ -22,6 +22,10 @@ type ThresholdVerifier struct {
 //   m<=t for unforgeability
 //   n-m>=t+1 for robustness
 func RandomBeaconThreshold(size int) int {
+	// avoid initializing the thershold to 0 when n=2
+	if size == 2 {
```

Comment:
what about:
```suggestion
	// avoid initializing the thershold to 0 when n<=2
	if size <= 2 {
```

It would also cover case where `size == 1`

Reply:
for `size = 1` the threshold is actually `0`. Moreover, the audit recommendation is not to run a threshold signature at all in this case, that's why this function is actually never called with `size==1`, that case is always treated before calling the function.